RecoCTPPS/PixelLocal/python/ctppsPixelLocalReconstruction_cff.py

```diff
@@ -1,15 +1,25 @@
 import FWCore.ParameterSet.Config as cms
 
 # geometry
-#from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+#from Geometry.VeryForwardGeometry.geometry_CTPPS_alaTotem_RECO_cfi import *
 
-
-# local track fitting
+# local clusterizer
 from RecoCTPPS.PixelLocal.ctppsPixelClusters_cfi import ctppsPixelClusters
 
 # local rechit producer
 from RecoCTPPS.PixelLocal.ctppsPixelRecHits_cfi import ctppsPixelRecHits
 
+# local track producer
+from RecoCTPPS.PixelLocal.ctppsPixelLocalTracks_cfi import ctppsPixelLocalTracks
+
+#ctppsPixelTracks = cms.EDProducer('CTPPSPixelLocalTrackProducer',
+#  patterFinderAlgorithm = cms.string('testPatternAlgorithm')
+#)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant